conf/application.conf

```diff
@@ -95,15 +95,6 @@ application.router = prod.Routes
 
 controllers {
 
-  # 300 is the default, you may need to change this according to your needs
-  confidenceLevel = 300
-
-  uk.gov.hmrc.mobileusercontact.controllers.MicroserviceHelloWorld = {
-    needsAuth = false
-    needsLogging = false
-    needsAuditing = false
```

Comment:
This reminds me - I think needsAuditing and needsLogging do still have an effect so we should add some for our new controllers.